### ISSUE TITLE
Error triggered on some files (.md, .npmignore, LICENCE)

### ISSUE THREAD

**Issue by jbdemonte** [zeit/pkg#75]
Trying to pack a project which includes jade (pug), these errors are triggered:

  > Error! Unexpected token (16:0)
    ../node_modules/jade/.npmignore
  > Error! Unexpected token (1:4)
    ../node_modules/jade/History.md
  > Error! Unexpected token (1:5)
    ../node_modules/jade/LICENSE

Build cmd: `pkg package.json --options expose-gc --targets node7-linux-armv7 --out-dir build`

jade dependency in package.json: `"jade": "^1.11.0",`

**Comment by igorklopov** [comment]
Hi! I failed to reproduce. Seems that you add `scripts` entry somewhere. Can you post full package.json? Or even better minimal example with all files to reproduce.

**Comment by jbdemonte** [comment]
Same project as in the other issue: https://github.com/jbdemonte/Retro-Manager

**Comment by igorklopov** [comment]
Please upgrade to `pkg@3.0.4` and try again

**Comment by carrotalan** [comment]
This is still an issue. Getting unexpected token when including pouchdb-fauxton in my project.

Command ran is:
`pkg app.js --targets node6.11.2-linux-x64`

Output is:
```
> Error! Unexpected token (14:0)
  /Volumes/Data/Users/work/git/kiosk-platform/kiosk-site-api/node_modules/pouchdb-fauxton/www/index.html
```

**Comment by dotarmin** [comment]
I get same error as @carrotalan when including pouchdb-fauxton, any update on this? 

Thanks!

/Armin

**Comment by tomachinz** [comment]
I see this while trying to use the library vue-golden-layout 

```
> pkg@4.3.4
> Error! Unexpected token (1:0)
  /Users/tom/Github/vue-golden-layout/node_modules/golden-layout/src/css/goldenlayout-base.css
```

**Comment by zhengxiaoyao0716** [comment]
It seems that `pkg` try to resolved non-javascript files (like `.css`) as javascript files and caused this problem.
For me, this just happend if the entrypoint was package.json or the project path, when I use the `.js` file as entrypoint, everything works well. I have two speculations for that: 
1. maybe the `pkg` ignored all the non-javascript files when we use `.js` file as entrypoint, but not ignored them when we used `package.json`. 
2. maybe when we use `package.json` or project path as entrypoint, `pkg` just try to bundle all the dependencies defined in the `package.json`, but when we used `.js` file, it would try to resolved the correctly dependencies that really used in the entrypoint files.
